plugins/jobs/src/js/components/form/GeneralFormSection.tsx

```diff
@@ -248,27 +184,25 @@ export default class GeneralFormSection extends React.Component<GeneralProps> {
             className="column-12"
             showError={showErrors && !!cmdErrors}
           >
-            <FieldLabel>
-              <FormGroupHeading required={formData.cmdOnly}>
-                <FormGroupHeadingContent title="Command" primary={true}>
-                  <Trans render="span">Command</Trans>
-                </FormGroupHeadingContent>
-                <FormGroupHeadingContent title="Command Info">
-                  <Tooltip
-                    content={cmdTooltipContent}
-                    interactive={true}
-                    maxWidth={300}
-                    wrapText={true}
-                  >
-                    <InfoTooltipIcon />
-                  </Tooltip>
-                </FormGroupHeadingContent>
-              </FormGroupHeading>
-            </FieldLabel>
             <FieldAutofocus>
-              <FieldTextarea
+              <Textarea
```

Comment:
same reason why we changed all the `FieldInput` to `TextInput`: we want to eventually have all components on ui-kit.

currently the ui has lots of "legacy" components that have been moved to ui-kit in the past. they look exactly the same and sometimes even have the same interface. i try to be a good citizen here and convert stuff as i touch it. 
whenever a component is reasonably close to being extincted i'll go all in and change all of the remaining occurrences. sometimes that's not easy because of past design decisions.

so TL;DR:
the boy scout approach :)